doc/api/tty.md

```diff
@@ -50,6 +50,14 @@ raw device. Defaults to `false`.
 added: v0.7.7
 -->
 
+Allows configuration of `tty.ReadStream` so that it operates as a raw device.
+
+For `stdin`, this mean that input is always available character-by-character
+(not including modifier characters such as `CTRL` or `ALT`). Additionally, all
```

Comment:
nit: I’d just call them “modifiers”, I don’t think they count as characters in any meaningful context